AccountManager/src/main/java/com/instinctools/trainerships/model/Client.java

```diff
@@ -0,0 +1,66 @@
+package com.instinctools.trainerships.model;
+
+import java.io.Serializable;
+import java.util.ArrayList;
+import java.util.List;
+
+import javax.persistence.CascadeType;
+import javax.persistence.Column;
+import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+import javax.persistence.JoinColumn;
+import javax.persistence.JoinTable;
+import javax.persistence.OneToMany;
+import javax.persistence.Table;
+
+@Entity
+@Table
+public class Client implements Serializable {
+
```

Comment:
In Account you set names for table and all fields - but for Client not
Lets follow some unified convention - better to add table and column names here also